tests/averaging/rotation/test_cycle_consistency.py

```diff
@@ -1,165 +0,0 @@
-"""Unit tests to ensure correctness of cycle error computation.
-
-Author: John Lambert
-"""
-from collections import defaultdict
-from typing import Dict, List, Tuple
-
-import numpy as np
-import pytest
-from gtsam import Rot3, Unit3
-
-import gtsfm.averaging.rotation.cycle_consistency as cycle_utils
-from gtsfm.evaluation.metrics import GtsfmMetricsGroup
-from gtsfm.two_view_estimator import TwoViewEstimationReport
-
-
-def test_compute_cycle_error_known_GT() -> None:
```

Comment:
making a note for myself that this test was renamed to `test_compute_cyclic_rotation_error()`